lib/workers/repository/dependency-dashboard.ts

```diff
@@ -245,6 +266,25 @@ export async function ensureDependencyDashboard(
 
   issueBody = appendRepoProblems(config, issueBody);
 
+  if (hasDeprecations) {
+    issueBody += '> ⚠ **Warning**\n> \n';
+    issueBody += 'The following dependencies are deprecated:\n\n';
+    issueBody += '| Datasource | Name | Replacement? |\n';
```

Comment:
```suggestion
    issueBody += '| Datasource | Name | Replacement PR? |\n';
```

Maybe this is clearer as it implies the Renovate PR. There may be a _manual_ replacement, that the user must do themselves.